### ISSUE TITLE
`Find legal facilities` button crashes EDDI when exploring deep space

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#1316]
## EDDI version in which issue is found
3.4.1-b1

## VoiceAttack version in which issue is found (as applicable)
1.7.5 - Steam

## Steps to reproduce
1. Explore outside of search range of EDSM cataloged systems
2. Click the `Find legal facilities` button

## Expected
EDDI continues to function and speaks "Not found within sensor range"

## Observed
EDDI crashes
```
2019-05-27T06:43:49 [Info] Logging:ExceptionHandler Reporting unhandled exception, anonymous ID f41f8aba-a465-4f34-9af5-a162e634f3ca:Newtonsoft.Json.JsonReaderException: Error reading JArray from JsonReader. Current JsonReader item is not an array: StartObject. Path '', line 1, position 1.
   at Newtonsoft.Json.Linq.JArray.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JArray.Parse(String json, JsonLoadSettings settings)
   at EddiStarMapService.StarMapService.GetStarMapSystemsCube(String starSystem, Int32 cubeLy, Boolean showEdsmId, Boolean showCoordinates, Boolean showInformation, Boolean showPermit)
   at EddiNavigationService.Navigation.GetServiceSystem(String serviceType, Int32 maxStationDistance, Boolean prioritizeOrbitalStations)
   at EddiNavigationService.Navigation.GetServiceRoute(String serviceType, Int32 maxStationDistance, Boolean prioritizeOrbitalStations)
   at EddiCrimeMonitor.ConfigurationWindow.<>c__DisplayClass5_0.<findIFRoute>b__0()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart() 
2019-05-27T06:43:49 [Info] EDDI:Stop EDDI 3.4.1-b1 stopped 
```

## Investigation
Suspected: EDSM may not have returned a valid empty JArray value.

**Comment by richardbuckle** [comment]
Fixed elsewhere for 3.4.1-b1

**Comment by richardbuckle** [comment]
@Tkael please re-test with the released 3.4.1-b1 because I believe this is fixed.

**Comment by Tkael** [comment]
Nope, still crashes... not fixed yet. PR updated.
